docs/tutorials/01_KalmanFilterTutorial.py

```diff
@@ -2,298 +2,195 @@
-#
-# which describes how the sensor (providing us with measurments) responds to the state
-# :math:`\textbf{x}_{k}` of the object (at time :math:`k`), with some noise :math:`\textbf{v}_{k}`
-# (since our measurements come with their own uncertainties).
-#
-#
-# We can rewrite:
+# Our goal is to infer :math:`\mathbf{x}_{k}`, given a sequence of measurements
+# :math:`\mathbf{z}_{1}, ..., \mathbf{z}_{k}`, (which we'll write as :math:`\mathbf{z}_{1:k}`). In
+# general :math:`\mathbf{z}` can include clutter and false alarms. We'll defer those complications
+# to later tutorials and assume, for the moment, that all measurements are generated by a target.
 #
-# .. math::
-#           \textbf{w}_{k} &\sim \textbf{H}_{k}(\eta_{1}, \eta_{2}, ...)\\
-#           \textbf{v}_{k} &\sim \textbf{Z}_{k}(\zeta_{1}, \zeta_{2}, ...)
-#
-# where :math:`\textbf{H}` and :math:`\textbf{Z}` are stochastic processes (something
-# demonstrating randomness in output) with parameters :math:`\eta` and :math:`\zeta`.
-# These functions generate instances of the noises represented by :math:`\textbf{w}` and
-# :math:`\textbf{v}`.
+# Prediction
+# ^^^^^^^^^^
 #
-# So,
+# We proceed under the Markovian assumption that :math:`p(\mathbf{x}_k) = \int_{-\infty} ^{\infty}
+# p(\mathbf{x}_{k-1}) p(\mathbf{x}_k|\mathbf{x}_{k-1}) d \mathbf{x}_{k-1}`, meaning that the
+# distribution over the state of an object at time :math:`k` can be predicted entirely from its
+# state at previous time :math:`k-1`. If our understanding of :math:`\mathbf{x}_{k-1}` was informed by a series of measurements up
+# to and including timestep :math:`k-1`, we can write
 #
 # .. math::
-#           \textbf{x}_{k|k-1} &\sim p(\textbf{x}_{k}\ |\ \textbf{x}_{k-1},\ \eta_{k})\\
-#               \textbf{z}_{k} &\sim p(\textbf{z}_{k}\ |\ \textbf{x}_{k},\ \zeta_{k})
+#           p(\mathbf{x}_k|\mathbf{z}_{1:k-1}) = \int_{-\infty}^{\infty} p(\mathbf{x}_{k-1}|\mathbf{z}_{1:k-1})
+#           p(\mathbf{x}_k|\mathbf{x}_{k-1}) d \mathbf{x}_{k-1}
 #
-# In other words, we concern ourselves with the probability distribution of the state and
-# measurement.
+# This is known as the *Chapman-Kolmogorov* equation. In Stone Soup we refer to this process as
+# *prediction* and to an object that undertakes it as a :class:`~.Predictor`. A predictor requires a
+# *state transition model*, namely a function which undertakes
+# :math:`\mathbf{x}_{k|k-1} = f(\mathbf{x}_{k-1}, \mathbf{w}_k)`, where :math:`\mathbf{w}_k` is
+# a noise term. Stone Soup has transition models derived from the :class:`~.TransitionModel` class.
 #
-# Then, adapting the law of total probability and Baye's rule:
-#
-# .. math::
-#           p(x) &= \int_{X} p(x|x')p(x')dx'\\
-#           P(X|Y) &= \frac{P(Y|X)P(X)}{P(Y)}
+# Update
+# ^^^^^^
 #
-# We get:
+# We assume a sensor measurement is generated by some stochastic process represented by a function,
+# :math:`\mathbf{z}_k = h(\mathbf{x}_k, \boldsymbol{\nu}_k)` where :math:`\boldsymbol{\nu}_k` is the noise.
```

Comment:
'\boldsymbol' requires a LaTeX package. Text appears in red on generated html page.

Reply:
Works ok for me. I guess not everyone gets amsfonts? Anyone else?

Could change symbol, leave unbold, make Roman...

Reply:
Looks okay on my system as well. I thought they were rendered by a javascript library, so shouldn't matter what packages you have locally.

Reply:
Weird, looks fine on mine now. Please ignore my comment